GitUI/UserControls/FileStatusList.cs

```diff
@@ -493,31 +514,6 @@ int GetLastIndex()
             }
         }
 
-        private static (Image Image, string Text) GetDisplayElements(ListViewItem item, PathFormatter formatter, int itemWidth)
-        {
-            var gitItemStatus = item.Tag<GitItemStatus>();
```

Comment:
I got NullReferenceException here, (called from line 1007), not investigating if this still occurs.

Reply:
Possbly fixed by e9020e67f3b0c175f216766f68aff1c269438c1e